src/utils/helpers.ts

```diff
@@ -69,69 +74,71 @@ export const getColor: GetColor = curry(
 // getFontFamily :: Family -> Props -> string
 // Family - any key of 'family' (src/theme/typography.ts)
 // Props - styled-components props object
-export const getFontFamily = curry(
+export const getFontFamily: HelperFn<keyof FontFamily, string> = curry(
   (family: keyof FontFamily, { theme }: ThemeType): string =>
     path(['typography', 'family', family], theme),
 );
 
 // getFontWeight :: Weight -> Props -> number
 // Weight - any key of 'weight' (src/theme/typography.ts)
 // Props - styled-components props object
-export const getFontWeight = curry(
+export const getFontWeight: HelperFn<keyof FontWeight, string> = curry(
   (weight: keyof FontWeight, { theme }: ThemeType): string =>
     path(['typography', 'weight', weight], theme),
 );
 
 // getFontSize :: Size -> Props -> string
 // Size - any key of 'size' (src/theme/typography.ts)
 // Props - styled-components props object
-export const getFontSize = curry(
+export const getFontSize: HelperFn<FontSize | string, string> = curry(
```

Comment:
Why are you using the union type here (`FontSize | string`)?  I don't think we allow passing any custom value here. Also if we would it's needed to write it as `... | (string & {})` to make suggestions work.

Reply:
```suggestion
export const getFontSize: HelperFn<keyof FontSize, string> = curry(
```